tests/driver_xbee/Makefile

```
@@ -1,6 +1,9 @@
 USEMODULE += xbee
 
 # No need of big buffer for this test
-GNRC_PKTBUF_SIZE=512
+ifndef CONFIG_GNRC_PKTBUF_SIZE
+  CONFIG_GNRC_PKTBUF_SIZE=512
+endif
+
```

This is the variable checked in `tests/driver_netdev_common/Makefile`.
```suggestion
ifndef GNRC_PKTBUF_SIZE
  GNRC_PKTBUF_SIZE=512
endif

```

---

Updated the file and skipped ifndef guards.